config/config.go

```diff
@@ -530,7 +530,7 @@ var (
 	UTF8NamesHeader      string         = model.EscapingKey + "=" + model.AllowUTF8
 
 	ScrapeProtocolsHeaders = map[ScrapeProtocol]string{
-		PrometheusProto:      "application/vnd.google.protobuf;proto=io.prometheus.client.MetricFamily;encoding=delimited",
```

Comment:
I'm not sure if I missed something, but is there any reason to change the proto message header?

It looks like an accident 🤔

Reply:
I've added a commit reverting it, please let me know if this should be kept!